simulator/opendaq-config.json

```diff
@@ -18,6 +18,7 @@
 	"ReferenceDevice":
 		{
 			"LocalId": "falselocalid",
+            "Serial": "falseserial",
```

Comment:
Should this be `SerialNumber` instead of `Serial`?

Reply:
I'd prefer SerialNumber as it matches the field name in DeviceInfo in that case.